Python/Product/PythonTools/PythonTools/Environments/AddExistingEnvironmentView.cs

```diff
@@ -52,13 +52,14 @@ ProjectView selectedProject
 
         public static IList<string> VersionNames { get; } = new[] {
             // we only support creating environments of offically supported versions
-            // see https://devguide.python.org/versions/
-            "3.7",
-            "3.8",
+            // see https://devguide.python.org/versions/            
             "3.9",
-            "3.10"
+            "3.10",
+            "3.11",
+            "3.12",
+            "3.13",
+            //todo: add 3.14 when it is released
```

Comment:
this was missed last update. Its not a common flow.. when adding an existing environment you can choose an existing path then it will attempt to auto select